src/main/scaffolds/stringPage/app/forms/$className$Form.scala

```diff
@@ -0,0 +1,24 @@
+package forms
+
+import play.api.data.{Form, FormError}
+import play.api.data.Forms._
+import play.api.data.format.Formatter
+
+object $className$Form extends FormErrorHelper {
+
+  def $className;format="decap"$Formatter(errorKeyBlank: String) = new Formatter[String] {
+
+    def bind(key: String, data: Map[String, String]) = {
+      data.get(key) match {
+        case None => produceError(key, errorKeyBlank)
+        case Some("") => produceError(key, errorKeyBlank)
+        case Some(s) => Right(s)
+      }
+    }
+
+    def unbind(key: String, value: String) = Map(key -> value)
+  }
+
+  def apply(errorKeyBlank: String = "error.required"): Form[String] =
```

Comment:
Since we're defining a new form here and it will always want the same value passed to this method, I'd recommend making it a `private val` inside the `object` that way, it can be changed in one place in code and you don't have to worry about changing it in all of the places it will be used in controllers/tests